config/routes.rb

```diff
@@ -28,8 +28,9 @@
     get :reschedule
   end
   resource :allocations, only: :show
-  resources :holidays, only: %i(index new create edit update) do
+  resources :holidays, only: %i(index new create edit) do
     post 'batch_create', on: :collection
```

Comment:
We could wrap all of these inside `collection do` and drop the `on: :collection` option.